caps/types.go

```diff
@@ -53,6 +71,14 @@ func (t *BoolFileType) Name() string {
 	return "bool-file"
 }
 
+// Error for reporting invalid/not-sanitized capabilities in places where we
+// expect everything to be valid.
+type notSanitizedError struct{}
+
+func (e *notSanitizedError) Error() string {
+	return "capability is not sanitized"
```

Comment:
I don't really understand what this is trying to say.. my guess is that other people likely won't as well.

It should also be a var:

    var errNotSanitized = errors.New("foo bar")

Or even inline, since there's really a single occurrence right now, which is never used in an error check.